--- webrtc/src/webrtc/sdk/sdk.gyp.orig	2016-08-08 17:13:11.000000000 +0900
+++ webrtc/src/webrtc/sdk/sdk.gyp	2016-08-08 17:13:14.000000000 +0900
@@ -254,6 +254,8 @@
             'objc/Framework/Headers/WebRTC/RTCMediaConstraints.h',
             'objc/Framework/Headers/WebRTC/RTCMediaStream.h',
             'objc/Framework/Headers/WebRTC/RTCMediaStreamTrack.h',
+            'objc/Framework/Headers/WebRTC/RTCMetrics.h',
+            'objc/Framework/Headers/WebRTC/RTCMetricsSampleInfo.h',
             'objc/Framework/Headers/WebRTC/RTCNSGLVideoView.h',
             'objc/Framework/Headers/WebRTC/RTCPeerConnection.h',
             'objc/Framework/Headers/WebRTC/RTCPeerConnectionFactory.h',
